### DISCUSSION TITLE
Safe to Uninstall Template Zero & One?

### DISCUSSION THREAD

**Discussion by sparkktv** [templaza/astroid-framework#1033] [Q&A]
Just curious if it is safe to uninstall Template Zero & One and just use Template Two? Seems weird they are all bundled together and not offered as separate downloads like JoomDev used to do. It would reduce the install size if they were separate and users had the option of what version they wanted and install it separate to the framework. That's why I ask if it's safe since they are bundled together.

**Comment by Chacapamac** [comment]
@sparkktv  — **It should be safe to uninstall any Astroid template.** 
I agree that it is cool to also have the templates as separated download. not sure including all three templates is really a problem.

**Comment by Giorgi625** [comment]
https://astroidframe.work/download they are available to download separately as well

**Comment by Chacapamac** [comment]
Thanks @Giorgi625 !👍
